C#/C#.sublime-syntax

```diff
@@ -1487,6 +1449,14 @@ contexts:
       scope: punctuation.section.braces.begin.cs
       set: initializer_constructor
 
+  function_call_begin_paren:
+    - meta_content_scope: meta.function-call.cs meta.group.cs
```

Comment:
This implementation causes `meta.group` to be applied to whitespace between function name and parens

```csharp
        Z<int> (P, x => { return 1; });
///           ^ meta.function-call.cs meta.group.cs
```

Reply:
well spotted good sir! fixed :)